scipy/interpolate/fitpack2.py

```diff
@@ -811,6 +811,9 @@ def __call__(self, x, y, dx=0, dy=0, grid=True):
             If `grid` is True: evaluate spline at the grid points
             defined by the coordinate arrays x, y. The arrays must be
             sorted to increasing order.
+            
+            Note that the axis ordering is inverted relative to
+            the output of meshgrid.
```

Comment:
Looks like the associated issue #3164 does call for some kind of documentation adjustment, but I'm not sure this doc addition within a private class object will be appropriately visible to end users.

Maybe the `interpolate` experts will correct me on that though.

Reply:
This is the documentation I was reading yesterday, so it is visible (the function is inherited by, e. g., RectBivariateSpline, that doc page links to __call__. Might be even better to specify what RBS returns somewhere in its documentation, but this is a start, and would have saved me several hours debugging yesterday.

Reply:
Thanks, ok, the change shows up here in the CI render of docs: https://6030-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.interpolate.RectBivariateSpline.__call__.html#scipy.interpolate.RectBivariateSpline.__call__

I hadn't looked looked at the docs for the `__call__` method itself. Maybe that will suffice, but I'll at least give some time for other to chime in.